src/frame/mod.rs

```diff
@@ -138,6 +216,34 @@ impl<'a> Request<'a> {
             Disconnect => Disconnect,
         }
     }
+
+    /// Get the [`FunctionCode`] of the [`Request`].
+    #[must_use]
+    pub const fn function_code(&self) -> FunctionCode {
+        use Request::*;
```

Comment:
```suggestion
        use Self::*;
```

Could also be replaced elsewhere for both `Request` and `Response`.

Reply:
This is not supported by the compiler. Or I'm doing something wrong 🤔 

```shell
error[E0432]: unresolved import `Self`
   --> src/frame/mod.rs:338:13
    |
338 |         use Self::*;
    |             ^^^^ `Self` cannot be used in imports
```